### PR TITLE
Release v1.23.0

### PR DESCRIPTION
**Pull request by jmrodri**
**Description of the change:**
Release v1.23.0 of `operator-sdk`

**Motivation for the change:**

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [x] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [x] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Review by everettraven — APPROVED** [review]
/lgtm

**Review by theishshah — APPROVED** [review]
/lgtm pending CI

**Comment by theishshah** [comment]
new /lgtm

File: changelog/generated/v1.23.0.md
Comment (asmacdo) [inline changelog/generated/v1.23.0.md:25]: + dropped olm support 0.19.1

File: changelog/generated/v1.23.0.md
Comment (asmacdo) [inline changelog/generated/v1.23.0.md:32]:
```suggestion
- Fix operator-sdk run bundle and upgrade bundle subCommands to allow them to work against Kubernetes versions < 1.19 and vendors like Openshift. ([#5973](https://github.com/operator-framework/operator-sdk/pull/5973))
```

**Review by asmacdo — APPROVED** [review]
some changelog nits, no need for re-review from me 
/lgtm

**Review by laxmikantbpandhare — APPROVED** [review]
/lgtm

one of the links is not accessible that's why failing with lint errors and I guess that will be fine.

**Pull request merged**
